.github/workflows/quality-checks.yaml

```diff
@@ -39,6 +40,9 @@ jobs:
       - name: Check out code...
         uses: actions/checkout@v2
 
+      - name: Configure Git to allow the current directory as safe
```

Comment:
Discussed offline. This does the job for now, and storing data on the root directory is not a big problem since we are using an isolated containers during the workflow execution. Resolving this conservation. Thank you, @jessefrn1 .